stairsplus/registrations.lua

```diff
@@ -74,3 +74,36 @@ for _, name in pairs(default_nodes) do
 		})
 	end
 end
+
+-- wool registrations
+
+local colorlist = {
+	{"white",      "White"},
+	{"grey",       "Grey"},
+	{"black",      "Black"},
+	{"red",        "Red"},
+	{"yellow",     "Yellow"},
+	{"green",      "Green"},
+	{"cyan",       "Cyan"},
+	{"blue",       "Blue"},
+	{"magenta",    "Magenta"},
+	{"orange",     "Orange"},
+	{"violet",     "Violet"},
+	{"brown",      "Brown"},
+	{"pink",       "Pink"},
+	{"dark_grey",  "Dark Grey"},
+	{"dark_green", "Dark Green"},
+}
+
+for i in ipairs(colorlist) do
```

Comment:
Since wool mod is opt-depend, you need to check for the presence of it at the start of this loop first.